catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/Nsili.java

```diff
@@ -24,6 +25,7 @@
 public interface Nsili {
 
     @GET
-    @Produces(MediaType.WILDCARD)
+    @Consumes({MediaType.TEXT_PLAIN, MediaType.APPLICATION_OCTET_STREAM, MediaType.WILDCARD})
```

Comment:
These changes were made to sync with another debugging effort that is not yet committed, and I pulled them in to ensure they were working for my test case.